tests/e2e/tutorials/tutorialBase.js

```diff
@@ -303,11 +319,10 @@ class TutorialBase {
     await this.takeScreenshot("closeStudy_after");
   }
 
-  async removeStudy() {
+  async removeStudy(studyId) {
     await this.takeScreenshot("deleteFirstStudy_before");
     try {
-      // wait for projects to be loaded
-      await utils.sleep(2000);
+      await this.waitForStudyUnlocked(studyId);
```

Comment:
why dont you pass the former sleep time (i.e. 2000) as the timeout of ``waitForStudyUnlocked``?

Reply:
Because it can take a bit longer closing the dynamic services. The default is 10".